firstvoices/backend/management/commands/_helper.py

```diff
@@ -85,9 +95,9 @@ def rebuild_index(index_name, index_document):
             bulk(es, dictionary_entry_iterator())
     except errors.BulkIndexError as e:
         # Alias configuration error
-        if "multiple indices" in str(e):
+        if "multiple documents" in str(e):
```

Comment:
This should be indices. alias points to an index, like "dictionary_entry" is an alias, that can point to an index that was created on x date with the name of "dictionary_entry_x". So we don't have to know actual names of indices.
This error can occur during a very less probable scenario when rebuilding indices.